bitnami/contour/values-production.yaml

```diff
@@ -59,6 +59,10 @@ configInline:
   leaderelection:
     # configmap-name: leader-elect
     configmap-namespace: '{{ .Release.Namespace }}'
+  ## This sets the service name that will be inspected for address details to be applied to Ingress objects.
+  # As by default the envoy services name deployed by this helm chart is 'contour-envoy', this configuration is needed
```

Comment:
I would change this here to remove any mention to 'contour-envoy' and optionally say that the expression used is the same one to generate the name in the Envoy-Service

https://github.com/bitnami/charts/blob/6bf9dfd96083702704461d268895a1e6416fdde3/bitnami/contour/templates/envoy/service.yaml#L4-L6

Reply:
Yes, done so.